lldb/packages/Python/lldbsuite/test/lldbplatformutil.py

```diff
@@ -300,6 +301,25 @@ def getCompilerVersion():
     return "unknown"
 
 
+def getWindowsVersion():
+    """Returns a string that represents the Windows version.
+
+    The string is a concatenation of the following, eparated by a dot:
+      - The major version number.
+      - The build number.
+
+    Example:
+      - Windows 11 version 24H2 -> "10.26100"
+      - Windows 10 version 1809 -> "10.17763"
+    """
+    import sys
+
+    if sys.platform != "win32":
+        return "unknown"
+    windows_version = sys.getwindowsversion()
```

Comment:
I was going to ask how this works for remote targets but I see that the same call was used before. So it works exactly as much as it did or did not previously.